server/src/services/metadata.service.ts

```diff
@@ -198,15 +198,15 @@ export class MetadataService extends BaseService {
     const dates = this.getDates(asset, exifTags, stats);
 
     const { width, height } = this.getImageDimensions(exifTags);
-    let geo: ReverseGeocodeResult, latitude: number | null, longitude: number | null;
-    if (reverseGeocoding.enabled && this.hasGeo(exifTags)) {
+    let geo: ReverseGeocodeResult = { country: null, state: null, city: null },
+      latitude: number | null = null,
+      longitude: number | null = null;
```

Comment:
Just for my own understanding: `null` here means that we'll actively delete the value (as opposed to `undefined`,  which would just be a no-change), but that only refers to the immich database, not the actual file EXIF?

Reply:
`null` is just the default of the variable and in this case means "not set"